cmd/server/assets/admin/realms/edit.html

```diff
@@ -23,20 +24,7 @@ <h1>Edit {{$realm.Name}}</h1>
     </p>
 
     <div class="card mb-3 shadow-sm">
-      <div class="card-header">
```

Comment:
no more link to realm-admin settings?

Reply:
mm nevermind. realm-admin as a thing is complex now.

Reply:
Yea, the permissions being non-binary makes some of our "helpful admin things" slightly less helpful.